code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -565,6 +605,105 @@ func (ssc *StorageSmartContract) GetChallengeHandler(ctx context.Context, params
 	return blobberChallengeObj.ChallengeMap[challengeID], nil
 }
 
+// statistic for all locked tokens of a stake pool
+func (ssc *StorageSmartContract) getStakePoolStatHandler(
+	ctx context.Context,
+	params url.Values,
+	balances cstate.StateContextI,
+) (interface{}, error) {
+	blobberID := datastore.Key(params.Get("blobber_id"))
+	if balances.GetEventDB() == nil {
+		return nil, errors.New("no event database found")
+	}
+
+	blobber, err := balances.GetEventDB().GetBlobber(blobberID)
+	if err != nil {
+		return nil, errors.New("blobber not found in event database")
+	}
+
+	delegatePools, err := balances.GetEventDB().GetDelegatePools(blobberID, int(stakepool.Blobber))
```

Comment:
it'll be difficult to map stakeppol.Blobber to dbs.Blobber when later someone checks the code
we should move to common constants here